Odata-docs/client/v7/query-options.md

```diff
@@ -11,6 +11,47 @@ ms.topic: article
 # Query options
 **Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v7.md)]
 
+There are mainly 2 ways to add Query Options to a DataServiceQuery
+1. Using AddQueryOption method.
+2. Using strongly typed C# LINQ query methods.
+
+## AddQueryOption Method
+The following example shows how to use to the `AddQueryOption` method to create a [DataServiceQuery&lt;TElement&gt;](/dotnet/api/microsoft.odata.client.dataservicequery-1) class implements the [IQueryable&lt;T&gt;](/dotnet/api/system.linq.iqueryable-1)
+
+```csharp
+DataServiceQuery peopleQuery = dsc.People
+    .AddQueryOption("$filter", "gender eq Female")
```

Comment:
Is gender Enum or String? Following from that, should it be `Female` or `'Female'`?

Reply:
Actually `Gender` is an `enum`. I have updated the sample code.
#Resolved